class_defines.py

```diff
@@ -1270,8 +1269,8 @@ def to_bezier(
         if set_startpoint:
             startpoint.co = locations[0].to_3d()
 
-        n = 2 * math.pi / angle
-        q = (4 / 3) * math.tan(math.pi / (2 * n))
+        n = FULL_TURN / angle
+        q = (4 / 3) * math.tan(HALF_TURN / (2 * n))
```

Comment:
I did a bit of algebra and it looks like it simplifies to  q= 4/3 tan(angle/4)
if so, I'll put it in the next PR

Reply:
I'm honestly not sure if such simplifications of formulas make a big performance improvement, i assume python optimizes some stuff anyway. But if it's not hurting readability it would be great of course.